inc/ix/ethdev.h

```diff
@@ -222,7 +222,7 @@ struct ix_rte_eth_rss_conf {
 #define ETH_RSS_IPV6_UDP    0x0080 /**< IPv6/UDP packet. */
 #define ETH_RSS_IPV6_UDP_EX 0x0100 /**< IPv6/UDP with extension headers. */
 /* Definitions used for redirection table entry size */
-#define ETH_RSS_RETA_NUM_ENTRIES 128
+#define ETH_RSS_RETA_NUM_ENTRIES 512
 #define ETH_RSS_RETA_MAX_QUEUE   16
```

Comment:
Changing the `#define` from 128 to 512 doesn't seem like the right thing to do. Remember, that IX must work both with ixgbe (which has 128 entries) and i40e (which has 512 entries). You should remove the `#define` and implement a different configuration scheme.